apps/stripe/src/app/api/saleor/payment-gateway-initialize-session/route.ts

```diff
@@ -2,79 +2,63 @@ import { withSpanAttributesAppRouter } from "@saleor/apps-otel/src/with-span-att
 import { compose } from "@saleor/apps-shared/compose";
 import { captureException } from "@sentry/nextjs";
 
-import { InitializeStripeSessionUseCase } from "@/app/api/saleor/payment-gateway-initialize-session/use-case";
-import { paymentGatewayInitializeSessionWebhookDefinition } from "@/app/api/saleor/payment-gateway-initialize-session/webhook-definition";
 import { appConfigPersistence } from "@/lib/app-config-persistence";
-import { UseCaseGetConfigError, UseCaseMissingConfigError } from "@/lib/errors";
 import { withLoggerContext } from "@/lib/logger-context";
 import { SaleorApiUrl } from "@/modules/saleor/saleor-api-url";
+import {
+  SaleorApiUrlCreateErrorResponse,
+  UnhandledErrorResponse,
+} from "@/modules/saleor/saleor-webhook-responses";
 
-const useCase = new InitializeStripeSessionUseCase({
+import { PaymentGatewayInitializeSessionUseCase } from "./use-case";
+import { paymentGatewayInitializeSessionWebhookDefinition } from "./webhook-definition";
+
+const useCase = new PaymentGatewayInitializeSessionUseCase({
   appConfigRepo: appConfigPersistence,
 });
 
 const handler = paymentGatewayInitializeSessionWebhookDefinition.createHandler(async (req, ctx) => {
-  /*
-   * todo create config repo
-   * todo: should we pass auth data to execute? likely yes
-   */
+  try {
+    /*
+     * todo create config repo
+     * todo: should we pass auth data to execute? likely yes
+     */
+
+    const saleorApiUrlResult = SaleorApiUrl.create({ url: ctx.authData.saleorApiUrl });
+
+    if (saleorApiUrlResult.isErr()) {
+      const response = new SaleorApiUrlCreateErrorResponse({
+        error: saleorApiUrlResult.error,
+      });
```

Comment:
I think this is too detailed, we are now returning this outside. I think it's fine to use here some generic response like "MalformedRequestError". We will know details in Sentry and this is scope of Stripe app only